odl/datasets/ray_transform.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Datasets for the ray transform."""
```

Comment:
Because tomography also means MRI, so I was planning to keep them separate. Is that a bad idea?